scripts/ci/jenkins_pipeline_csi

```diff
@@ -1,4 +1,12 @@
 pipeline {
+    environment {
+        registryCredential = 'csi_w3_user'
```

Comment:
`registryCredentialsID` is more accurate

Reply:
done